ui/index.js

```diff
@@ -59,7 +59,7 @@ async function startApp (metamaskState, backgroundConnection, opts) {
   })
 
   // if unconfirmed txs, start on txConf page
-  const unapprovedTxsAll = txHelper(metamaskState.unapprovedTxs, metamaskState.unapprovedMsgs, metamaskState.unapprovedPersonalMsgs, metamaskState.unapprovedTypedMessages, metamaskState.network)
+  const unapprovedTxsAll = txHelper(metamaskState.unapprovedTxs, metamaskState.unapprovedMsgs, metamaskState.unapprovedPersonalMsgs, metamaskState.unapprovedDecryptMsgs, metamaskState.unapprovedEncryptionPublicKeyMsgs, metamaskState.unapprovedTypedMessages, metamaskState.network)
```

Comment:
will these messages be network dependent?

Reply:
No, they are not about transactions. But what do you mean? If I will change the network on `testnet`, then will it count by another handler? I didn't check how it works right now, but if each network has own global counter over all requests, then my changes will inherit the same behavior.

Reply:
This `txHelper` function only counts the unapproved transactions for the current network - that's why the network is passed in. The changes made here aren't affected by the network. The same goes for the other types of signing.